llvm/test/CodeGen/AMDGPU/llvm.log.ll

```diff
@@ -3076,7 +3076,7 @@ define float @v_log_f32_fast(float %in) {
   ret float %result
 }
 
-define float @v_log_f32_unsafe_math_attr(float %in) "unsafe-fp-math"="true" {
```

Comment:
I believe this file already has test for the flag, this test function is specifically to test the attributes so it can be deleted (same applies for most of these files)